Code/Framework/AzToolsFramework/AzToolsFramework/Thumbnails/ThumbnailerComponent.cpp

```diff
@@ -26,6 +26,7 @@ namespace AzToolsFramework
             : m_missingThumbnail(new MissingThumbnail())
             , m_loadingThumbnail(new LoadingThumbnail())
             , m_currentJobsCount(0)
+            , m_maxThumbnailJobs(AZStd::thread::hardware_concurrency())
```

Comment:
I think this requires a minimum count in case the function returns 0.
AZStd::max(1, AZStd::thread::hardware_concurrency())

Reply:
I meant to do this! Thanks, Guthrie, change made.